pyproject.toml

```diff
@@ -38,6 +38,7 @@ pandas = "^2.2.3"
 python-docx = "^1.1.2"
 marker-pdf = { version = "^1.6.2", optional = true }
 dataclasses-json = "^0.6.7"
+openpyxl = "^3.1.5"
```

Comment:
If you're doing this because of pandas needing it. I'd recommend using `pandas[excel]` instead of this. I think the poetry syntax for extras like this is a little different though, worth checking